### PR TITLE
Fixed netlify builds, updated graphql to v15, bumped node version

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,11 +91,11 @@
     "urql": "^1.9.7"
   },
   "devDependencies": {
```
Comment (jonkoops) [inline package.json:93]: Is Dependabot not finding any updates for these?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -22,3 +22,5 @@ update_configs:
           dependency_name: "marked"
       - match:
           dependency_name: "body-scroll-lock"
+      - match:
+          dependency_name: "@svgr/webpack"
```
[the remark this answers is not in the document]
Comment (chrisvanmook) [reply .dependabot/config.yml:26]: was suspecting this package but still seems to break..

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-v8.15.0
+v12.16.3
```
Comment (jonkoops) [inline .nvmrc:1]: Let's use the v12 LTS instead of a fixed version here.

```suggestion
lts/erbium
```

Comment (chrisvanmook) [reply .nvmrc]: nice! I was looking for this, but couldn't find it. Where did you got this from?

**Pull request merged**